CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ### Added
+- Screenshot attaching on failure
+### Added
```

Comment:
Each section must be mentioned once.
Refer https://keepachangelog.com/en/1.0.0/